src/pdf/pdf.ts

```diff
@@ -549,16 +549,11 @@ export class PDF_ extends ExtendedPDF {
 
   private _renderQRCode(): void {
 
-    const qrcode = generateQRCode(this._data, utils.mm2pt(46));
-
+    const qrcode = generateQRCode(this._data, "pdf", utils.mm2pt(67), this._marginTop + utils.mm2pt(17), utils.mm2pt(46));
 
     //-- Add QR Code
 
-    this.addPath(qrcode, utils.mm2pt(67), this._marginTop + utils.mm2pt(17))
-      .undash()
```

Comment:
`.undash()` is still needed for the border of the Swiss cross overlay. But it should be implemented there and not on the qr code:

```diff
this.addPath(swissCrossBackground, utils.mm2pt(86.5), this._marginTop + utils.mm2pt(36))
+      .undash()
      .fillColor("black")
      .lineWidth(1.42)
      .strokeColor("white")
      .fillAndStroke();
```